code/__HELPERS/game.dm

```diff
@@ -350,7 +350,7 @@
 /proc/flick_overlay(image/I, list/show_to, duration)
 	for(var/client/C in show_to)
 		C.images += I
-	addtimer(CALLBACK(GLOBAL_PROC, /.proc/remove_images_from_clients, I, show_to), duration, TIMER_CLIENT_TIME)
+	addtimer(CALLBACK(GLOBAL_PROC, /proc/remove_images_from_clients, I, show_to), duration, TIMER_CLIENT_TIME)

```

Comment:
I feel like this is just unnecessary and is more of a style choice. Why does this matter?

Reply:
It's not immediately obvious whether the `.` in `/.proc` actually does anything at all and its meaning is context-dependent: `/.proc` is the same as `/proc` but `/datum/.proc` is the same as `/datum.proc`.

This style issue is duplicated elsewhere in the codebase but in ways that did not happen to cause problems for SpacemanDMM. I since had to fix the parse for `/.proc` for other reasons so if you care I can undo these.

Reply:
oranges would probably call it needless churn but I don't mind too much. I'd lean more towards not liking it but eh whatever.

Reply:
actualy no for the purpose of searching the code for proc uses this is better, yeah keep this.